src/reports/repo/utils.py

```diff
@@ -23,6 +23,7 @@
     _get_pkg_meta
 )
 from .models import Arch, Graph, Image, PackageMetaType, Repo
+from .rpmmd import RepoSack
```

Comment:
that we've had since forever :)
And the changes below are a switch to the Package api
Possibly mention this in the commit msg?